devsetup/scripts/ipv6-nat64/README.md

```diff
@@ -1,6 +1,22 @@
 Development environment - IPv6 NAT64 lab
 ========================================
 
+Prerequisites
+-------------
+
+Network setup relies on `systemd-resolved` service enabled and used for DNS
+resolution. If it's not installed on your machine (for example, if you use
+RHEL), then:
+
+#. Install the package: `dnf install -y systemd-resolved`
+#. Enable and start the service: `systemctl enable systemd-resolved && systemctl start systemd-resolved`
```

Comment:
could use systemctl enable --now systemd-resolved instead to start along with enable